### DISCUSSION TITLE
[I1] no aparezco en la hoja con las salas

### DISCUSSION THREAD

**Discussion by MagnaThyrsus** [IIC2133-PUC/2024-2#133] [Ayuda]
Lo que dice el título. Soy sección 2 y mi RUT no aparece. Se que es tarde para preguntar pero comprenderán que uno no espera que uno simplemente no exista en la lista de salas para la interrogación de un ramo. Mi RUT parte con 19.685... Por otro lado soy PIANE, no sé si eso tenga que ver.

**Comment by tomastrivino** [comment]
Hola,
si eres PIANE deberías rendir la prueba en la sala correspondiente a PIANE. Te llegó un correo con información?

Saludos

**Reply by MagnaThyrsus** [reply]
Efectivamente llego uno ayer, pero fue movido a spam automaticamente por alguna razón. Gracias por aclarar.

**Reply by tomastrivino** [reply]
Hola,
Te informo que la sala PIANE es la B22. Se cambió recientemente

Saludos
